init.sql

```diff
@@ -232,6 +239,8 @@ CREATE TABLE IF NOT EXISTS person (
     PRIMARY KEY (id)
 );
 
+ALTER TABLE person ADD COLUMN IF NOT EXISTS ethnicity_id SMALLINT REFERENCES ethnicity(id) NOT NULL DEFAULT 1;
```

Comment:
I'll make this change when I merge the PR:

* Move this to the bottom of the `init.sql`.